xwayland/xwm.h

```diff
@@ -0,0 +1,84 @@
+#ifndef XWAYLAND_INTERNALS_H
+#define XWAYLAND_INTERNALS_H
+#include <wayland-server-core.h>
+#include <wlr/xwayland.h>
+
+/* wlc's atom list:
```

Comment:
Will remove that once we're handling a few more events, it's useful to have right now.